app/enterprise/2.1.x/developer-portal/configuration/authentication/basic-auth.md

```diff
@@ -2,56 +2,76 @@
 title: Enable Basic Auth in the Dev Portal
 ---
 
-### Introduction
+## Introduction
 
-The Kong Developer Portal can be fully or partially authenticated using HTTP protocol's Basic Authentication scheme. Requests will be sent with the Authorization header that
+The Kong Developer Portal can be fully or partially authenticated using HTTP protocol's Basic Authentication scheme. Requests are sent with an Authorization header that
 contains the word `Basic` followed by the base64-encoded `username:password` string.
 
-Basic Authentication for the Dev Portal can be enabled in three ways:
+Basic Authentication for the Dev Portal can be enabled using any of the following ways:
 
-- via the [Kong Manager](#enable-basic-auth-via-kong-manager)
-- via the [the command line](#enable-basic-auth-via-the-command-line)
-- via the [the Kong configuration file](#enable-basic-auth-via-the-kong-conf)
+- [Kong Manager](#enable-basic-auth-via-kong-manager)
+- [Command line](#enable-basic-auth-via-the-command-line)
+- [Kong configuration file](#enable-basic-auth-via-the-kong-conf)
+
+**Warnings:**
+
+- Enabling authentication in the Dev Portal requires use of the
+Sessions plugin. Developers will not be able to log in if this is not properly set.
+For more information, see
+[Sessions in the Dev Portal](/enterprise/{{page.kong_version}}/developer-portal/configuration/authentication/sessions).
+
+- When Dev Portal Authentication is enabled, content files remain unauthenticated until a role is applied to them. The exceptions are `settings.txt` and `dashboard.txt` that begin with the `*` role. For more information, see
+[Developer Roles and Content Permissions](/enterprise/{{page.kong_version}}/developer-portal/administration/developer-permissions).
 
->**Warning** Enabling authentication in the Dev Portal requires use of the
-> Sessions plugin. Developers will not be able to login if this is not set
-> properly. More information about [Sessions in the Dev Portal](/enterprise/{{page.kong_version}}/developer-portal/configuration/authentication/sessions)
 
 ### Enable Portal Session Config
 
 In the Kong configuration file, set the `portal_session_conf` property:
 
-```
-portal_session_conf={ "cookie_name": "portal_session", "secret": "CHANGE_THIS", "storage": "kong" }
+```bash
```

Comment:
This isn't really bash - you're editing `kong.conf`, which I guess you might do using bash, but you're just as likely to use a text editor. Same goes for the next two examples. 

I'd stick with no syntax highlighting here. `kong.conf` is a mix of formats - technically the latter part of the session config is in json, but the property name is not.

Reply:
Done in commit eefd763